### PR TITLE
do not log name "mod_appdynamics"

### PR DESCRIPTION
**Pull request by SaschaBrechmannVHV**
Change Logoutput to "mod_opentelemetry_wbserver_sdk"

### CODE REVIEW

File: instrumentation/otel-webserver-module/src/apache/ApacheTracing.cpp [hunk 1]
```diff
@@ -47,7 +47,7 @@ void ApacheTracing::writeTrace(server_rec* s, const char* funcName, const char*
 
     if (m_state == TRACE_AS_ERROR && s)
     {
-        ap_log_error(APLOG_MARK, APLOG_ERR, 0, s, "mod_appdynamics: %s: %s", funcName, note);
+        ap_log_error(APLOG_MARK, APLOG_ERR, 0, s, "mod_opentelemetry_wbserver_sdk: %s: %s", funcName, note);
```
Comment (DebajitDas) [inline instrumentation/otel-webserver-module/src/apache/ApacheTracing.cpp:50]: Thanks for raising this.
Spelling mistake ?
wbserver ---> webserver

**Review by DebajitDas — CHANGES_REQUESTED** [review]

File: instrumentation/otel-webserver-module/src/apache/ApacheTracing.cpp (hunk 1 above)
Comment (SaschaBrechmannVHV) [reply instrumentation/otel-webserver-module/src/apache/ApacheTracing.cpp]: Yes, needs to read my patch one more time ...

**Review by SaschaBrechmannVHV — COMMENTED** [review]
Typo changed

**Review by DebajitDas — APPROVED** [review]
LGTM

**Review by kpratyus — APPROVED** [review]

**Pull request merged**